code/game/machinery/cryopod.dm

```diff
@@ -743,8 +743,7 @@
 		target_cryopod = safepick(free_cryopods)
 		if(target_cryopod.check_occupant_allowed(person_to_cryo))
 			var/turf/T = get_turf(person_to_cryo)
-			var/obj/effect/portal/SP = new /obj/effect/portal(T, null, null, 40)
-			SP.name = "NT SSD Teleportation Portal"
+			var/obj/effect/portal/cryo = new /obj/effect/portal(T, null, null, 40)
```

Comment:
the last bit after a slash in the var, is the name, you'll need something like var/obj/effect/portal/cryo/cryo_portal 
```suggestion
			var/obj/effect/portal/cryo/cryo_portal = new /obj/effect/portal/cryo(T, null, null, 40)
```

Reply:
Redone in what I was told is a better way